sycl/include/sycl/detail/vector_convert.hpp

```diff
@@ -0,0 +1,330 @@
+//==-- vector_convert.hpp --- vec::convert implementation ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <sycl/detail/generic_type_traits.hpp> // for is_sigeninteger, is_s...
+
+#ifndef __SYCL_DEVICE_ONLY__
+#include <sycl/builtins_scalar_gen.hpp> // for ceil, floor, rint, trunc
+
+#include <cfenv> // for fesetround, fegetround
+#endif
+
+#include <type_traits>
+
+namespace sycl {
+
+enum class rounding_mode { automatic = 0, rte = 1, rtz = 2, rtp = 3, rtn = 4 };
```

Comment:
Shouldn't this be inside `_V1`?

Reply:
Good catch, I will submit a follow-up PR